### ISSUE TITLE
Don't validate when showing information

### ISSUE THREAD

**Issue by TAGC** [natemcmaster/CommandLineUtils#32]
I'm trying to create a command line application with a single required argument ("version"):

```cs
var versionArgument = app.Argument("version", "The version to set the project as.").IsRequired();
```

As expected, I get an error message when trying to run my program without specifying this argument:

```
$ dotnet bin/Release/netcoreapp2.0/dotnet-setversion.dll
The version field is required.
Specify --help for a list of available options and commands.
```

However, I *don't* expect to get this error when displaying information (help or version), but that's what currently happens:

```
$ dotnet bin/Release/netcoreapp2.0/dotnet-setversion.dll --help
Usage: dotnet-setversion [arguments] [options]

Arguments:
  version  The version to set the project as.

Options:
  -h|--help  Show help information
  -f|--file  Identifies the csproj file to update. Looks in the current working directory if not set.
The version field is required.
Specify --help for a list of available options and commands.
```

**Comment by natemcmaster** [comment]
Yup, this is a bug. It should be fixed in the next patch release, 2.1.1. In the meantime, you can validate this fix in with this build:

https://www.myget.org/feed/natemcmaster/package/nuget/McMaster.Extensions.CommandLineUtils/2.1.1-alpha.126
https://ci.appveyor.com/project/natemcmaster/commandlineutils/build/2.1.1-alpha.126/artifacts

**Comment by TAGC** [comment]
Yeah seems to work great, thanks.
